index.php

```diff
@@ -140,6 +158,20 @@ function form($instance) {
 	}
 }
 
+/*
+ * Cascading Style Sheets   
+ */
+
+function add_herrnhuter_stylesheet( )
+ {
+  wp_register_style( HHL . 'StyleSheets', plugins_url( 'css/styles.css', __FILE__ ) );
```

Comment:
If we include the CSS file by default, this would be a BC break (as currently all sites using the plugin have added their own CSS.) My suggestion is to make this optional (a checkbox in the widget that is off by default, or on only on new installations).

Reply:
I think this would not be a BC break.  User defined style sheet should be loaded later and override these settings.

If the user has done it right he /s he has created a child-theme. These styles are applied last.